src/interfaces/interfaces.ts

```diff
@@ -0,0 +1,13 @@
+import {PieceColor, PieceState} from "../utils/consts/piece";
+
+export interface IPiece {
+    state: PieceState,
+    color: PieceColor,
+}
+
+export interface ICell {
+    row: number;
+    col: number;
+    isBlackCell: boolean;
```

Comment:
I think it's better to have a property `color` and values like `"black" | "white"`. But of course there is not going to be any other colors, so it's up to you

Reply:
I'm checking either I should render black cell or not. So regardless of color I should check should I render cell where I can place piece or not

Reply:
Maybe it should be renamed to isCell? Because if not I'll just skip this and won't render a cell

Reply:
up to you